modules/ROOT/pages/about-my-applications.adoc

```diff
@@ -12,14 +12,30 @@ image::ex2-my-applications.png[]
 A contract holds information about the contract with a client ID and client secret for the API.
 Each application lists a description, its URL, redirect URIs, grant types, and a usage dashboard.
 
-The usage information lets you set the verification duration. If not specified, the range is the last 8 days. The usage lists the total number of requests to the API, average latency, the error rate, and graphs for total requests, average latency, requests by HTTP status code, and the error percentage.
-
-The applications result when users click Request Access to establish
-a link between an application and an API. Either the application already exists,
-or the user creates the application from the Request Access menu.
+The usage information enables you to set the verification duration. If not specified,
+the range is the previous eight days. The usage lists the total number of requests to
+the API, average latency, the error rate, and graphs for total requests,
+requests by HTTP status code, and the error percentage.
 
 Applications in My Applications are registered using API Manager in Anypoint Platform.
 
+To implement an API, the client application must first request access. When the
```

Comment:
Incorrect statement. API can be created regardless of having one, many or no contracts. I would say instead of "To implement an API" something like "To register a client application to an existing API".

Reply:
Fixed. Thank you!